components/button/stories/button.stories.js

```diff
@@ -58,6 +58,13 @@ export default {
 		isActive,
 		isPending,
 		staticColor,
+		noWrap: {
+			name: "Disable Label Wrap",
+			type: { name: "boolean" },
+			table: {
+				category: "Content",
```

Comment:
```suggestion
				category: "Advanced",
```
I would categorize this as advanced, since it's not a regular option from the spec and isn't typically recommended. Otherwise it would be in the "Component" category. "Content" is typically for controls that are for modifying the content displayed within the story (e.g. text, image source, icon name, etc).